dags/sql/select-sensor_id_count_lookback.sql

```diff
@@ -31,7 +45,13 @@ SELECT
     'Pass threshold: ' || to_char(
             FLOOR({{ params.threshold }}::numeric * AVG(lb.lookback_count)),
             'FM9,999,999,999'
-            ) AS passing_value
+            ) AS passing_value,
```

Comment:
the to_char function does a round up to 0 sig fig, i think we should include at least 1 decimal

Reply:
I floored all the thresholds because the `today_count` will always be an integer (Row count / count distinct ids) so I thought it makes more sense to compare int/int.

Reply:
make sense :meow_salute: